js/services/ViewDataFormatterService.js

```diff
@@ -464,7 +464,7 @@ class ViewDataFormatterService {
             labels.push({
                 text: '⚡ Auto-Create',
                 cssClass: 'queue-tag tag-auto-create',
-                title: 'Auto Queue Creation Enabled',
+                tooltip: 'Auto Queue Creation. When auto-creation is enabled on a YARN Capacity Scheduler parent queue, it dynamically generates child leaf queues based on user or group mappings, facilitating flexible and scalable resource allocation without manual configuration. These auto-created queues inherit properties from their parent and can be automatically deleted after a period of inactivity, typically 300 seconds, unless configured otherwise.',
```

Comment:
> inherit properties from their parent

Nit: inherit properties from their parent and the parent's auto-creation template configurations

Reply:
thx
<img width="509" alt="image" src="https://github.com/user-attachments/assets/7180d618-972e-4a1a-940e-511981c738f7" />